benchmark/benchmark.py

```diff
@@ -0,0 +1,232 @@
+import argparse
+import bz2
+import csv
+import datetime
+import json
+import os
+import sys
+from io import StringIO
+from pathlib import Path
+
+from matplotlib import pyplot as plt  # type: ignore
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.dirname(SCRIPT_DIR))
+
+from eyecite import clean_text, get_citations
+
+csv.field_size_limit(sys.maxsize)
+
+root = Path(__file__).parent.absolute()
+
+MAX_ROWS_IN_MD = 51
+
+
+class Benchmark(object):
+    """Benchmark the different eyecite branches"""
+
+    def get_filepath(self, filename):
+        return Path.joinpath(root, filename)
+
+    def generate_branch_report(self, branch: str) -> None:
+        """Generate Report
+
+        :param branch: Is a branch from main or not
+        :return: None
+        """
+        with bz2.BZ2File(self.get_filepath("bulk-file.csv.bz2")) as zipfile:
+            csv_data = csv.DictReader(zipfile.read().decode(), delimiter=",")
+        count = 0
+        now = datetime.datetime.now()
+        data = []
+        for row in csv_data:
+            id = row.pop("id")
```

Comment:
I think you can drop this line too. No need to pop it, and `id` is a reserved word in Python we shouldn't be using anyway. This was just to make the `values()[0]` trick work, anyway, right?

Reply:
the cluster Id is used in the output to let people review the source material

Reply:
but dropped it and used row['id']

Reply:
Yep. :)